### PR TITLE
5680 members notes

### PR DESCRIPTION
**Pull request by linesarefuzzy**

### CODE REVIEW

File: app/assets/javascripts/backbone/views/notable_view.coffee [hunk 1]
```diff
@@ -1,6 +1,4 @@
-class MS.Views.OrganizationView extends Backbone.View
-
-  el: '.coop'
```
Comment (hooverlunch) [inline app/assets/javascripts/backbone/views/notable_view.coffee]: Hooray for removing explicit `el`s!

File: app/assets/javascripts/backbone/views/person_view.coffee [hunk 2]
```diff
@@ -3,6 +3,7 @@ class MS.Views.PersonView extends Backbone.View
 
   initialize: ->
     @showHideUserFields()
+    new MS.Views.NotableView({ el: @el })
```
Comment (hooverlunch) [inline app/assets/javascripts/backbone/views/person_view.coffee]: Braces and spaces not needed :)

File: app/views/admin/organizations/show.html.slim [hunk 3]
```diff
@@ -33,21 +33,13 @@ section.coop.details
...
-        = render partial: 'admin/notes/note', locals: { note: @new_note }
+  section.block
+    = render "admin/notes/notes_section"
 
 = render "admin/media/modal"
 
 javascript:
   $(function() { 
-    new MS.Views.OrganizationView();
+    new MS.Views.NotableView({ el: 'coop' });
```
Comment (hooverlunch) [inline app/views/admin/organizations/show.html.slim]: Should there be a `.` before `coop`? I'm curious how this is even working currently... Please be sure to test all concrete instantiations when you abstract something out like this...

Comment (hooverlunch) [inline app/views/admin/organizations/show.html.slim]: Or is there a new `<coop>` tag? That would be sweet!

Comment (linesarefuzzy) [inline app/views/admin/organizations/show.html.slim]: Huh. Yup, that's a typo. But it works. No idea how.

**Pull request merged**